pitt-traditions-arduino-ws/pitt-traditions/pitt-traditions.ino.ino

```diff
@@ -0,0 +1,275 @@
+/**************************************************************
+ * AUTHOR(S):    Morgan Visnesky
+ * DATE:         03/18/2022
+ * FILENAME:     pitt-traditions.ino

```

Comment:
@cmg137 you had created a whole new file when making this change.  The file you added has an invalid file extension.